public-ui/src/app/components/db-no-results/db-no-results.component.ts

```diff
@@ -84,7 +85,8 @@ export class DbNoResultsComponent implements OnChanges, OnDestroy {
   }
 
   public searchDomains(query: string) {
-    this.subscriptions.push(this.api.getDomainSearchResults(query).subscribe(results => {
+    this.subscriptions.push(this.api.getDomainSearchResults(query,
+      environment.publicUiUrl + this.router.url).subscribe(results => {
```

Comment:
I don't think including the full URL is necessary here, seems like just the url should do

Reply:
![image](https://user-images.githubusercontent.com/21062724/55755603-0ec33d00-5a15-11e9-84b6-9f5b903e561f.png)
Was adding it as helen asked for this. I agree with not needing this though since there is a way to view events seperated by url by selecting the project on the top.

Reply:
I see the concern, the event category should be enough here, but if you really want the page could you just use `window.location.href` instead of the environment flag?

Reply:
Changed it to use window.location.href instead. thanks.